site/index.html

```diff
@@ -1,14 +1,102 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <title>Your Website Name</title>
-    <link rel="stylesheet" href="/assets/css/reset.css" type="text/css" />
-    <link rel="stylesheet" href="/assets/css/global.css" type="text/css" />
-    <link rel="stylesheet" href="styles.css" type="text/css" />
-  </head>
-  <body>
-    <nav><a href="/">Home</a><a href="/about">About</a></nav>
-    <h1>Homepage</h1>
-  </body>
+
+<head>
+  <meta charset="UTF-8" />
+  <title>Welcome to Boston Arts Academy</title>
+  <link rel="stylesheet" href="/assets/css/reset.css" type="text/css" />
+  <link rel="stylesheet" href="/assets/css/global.css" type="text/css" />
+  <link rel="stylesheet" href="styles.css" type="text/css" />
+</head>
+
+<body>
+  <nav><a href="/">Home</a><a href="/about">About</a></nav>
+  <div class="header-box">
+    <h1>Artist-scholars designing their own future.</h1>
+    <div>view their wokr</div>
+  </div>
+  <div class="upcoming events">
```

Comment:
I think you meant for class to be `upcoming-events`? When you put a space that means it's another class.